### ISSUE TITLE
dropout need to set mask's device same as input Tensor!

### ISSUE THREAD

**Issue by XIronMan0220** [wavefrontshaping/complexPyTorch#24]

**Comment by wootommy** [comment]
Meet the same problem!

```sh
... line 106, in complex_dropout
    return mask*input
RuntimeError: Expected all tensors to be on the same device, but found at least two devices, cuda:0 and cpu!
```

**Comment by wavefrontshaping** [comment]
complexPyTorch is obsolete and not maintained as since version 1.7 PyTorch natively supports complex numbers.

**Comment by XIronMan0220** [comment]
> complexPyTorch is obsolete and not maintained as since version 1.7 PyTorch natively supports complex numbers.

But many modules in PyTorch is still not support complex numbers.

**Comment by wavefrontshaping** [comment]
My bad, I have to admit I did not touch PyTorch for a while, I assume the integration of complex numbers meant all was implemented.
What is missing?

**Comment by wavefrontshaping** [comment]
Should be solved now (commit 7f611aa5760bc86f43e11721a1f8d9dc09ef699a)
